### DISCUSSION TITLE
Budibase on Digital Ocean: Row limit when using Budibase DB?

### DISCUSSION THREAD

**Discussion by farmserve** [Budibase/budibase#6573] [General]
Hi there, I couldn't find any documentation on this question, but I was wondering if I self-host Budibase using Digital Ocean and use Budibase DB exclusively as my backend, will there be a limit to how many rows I can store in the backend?

Thank you in advance for your replies.

**Comment by mike12345567** [comment]
Hi @farmserve - if you self host Budibase in any format then there will be no limits on what you can do with it.

**Reply by farmserve** [reply]
Fantastic thank you
